block_test.go

```diff
@@ -20,7 +19,9 @@ func TestBlockParsing(t *testing.T) {
 	}
 
 	c := blk.Cid()
-	fmt.Println(c)
+
+	_ = c
+	// f_ = cmt.Println(c)
```

Comment:
We should probably be either using this value or not generating it. What's this test for?